operator-pipeline-images/operatorcert/github.py

```diff
@@ -65,7 +65,6 @@ def get(
     """
     session = _get_session(auth_required=auth_required)
     LOGGER.debug("GET GitHub request url: %s", url)
-    LOGGER.debug("GET GitHub request params: %s", params)
```

Comment:
Is there any reason for removing the debug log?

Reply:
Nope, misunderstanding, thanks for reminding